app/controllers/statistics_controller.rb

```diff
@@ -0,0 +1,18 @@
+class StatisticsController < ApplicationController
+  def metric_percentage
+    count_metric_snapshot = MetricSnapshot.where(code: params[:metric_code]).count
+    total_configuration = MetricConfiguration.all.count
```

Comment:
What do you think about using `MetricConfiguration.count` here? This way the count is done through SQL on database instead of retrieving the whole data and them asking ruby to count the array. Just a performance concern.